src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -125,11 +143,20 @@ export class DotEditLayoutComponent implements OnInit {
         this.showTemplateLayoutSelectionDialog = false;
     }
 
+    private setUpLayOut(pageView: DotPageView): void {
+        this.pageView = pageView;
+        this.templateContainersCacheService.set(this.pageView.containers);
+        this.initForm();
+    }
+
     private initForm(): void {
         this.form = this.fb.group({
             title: this.isLayout() ? null : this.pageView.template.title,
             layout: this.fb.group({
-                body: this.pageView.layout.body || {},
+                body:
+                    this.dotEditLayoutService.getDotLayoutBody(
```

Comment:
Why is this change for?

Reply:
this gets the data that comes from the service and change the format to the result that will be send,  basically not wait the user to make changes but instead make it on load. and be able to compare if form is pristine